xtask/src/ci/target_matrix.rs

```diff
@@ -1,18 +1,37 @@
+use std::env;
+use std::process::Command;
+
 use clap::builder::BoolishValueParser;
 use clap::Parser;
-use serde::Serialize;
+use cross::{config::bool_from_envvar, shell::Verbosity, CommandExt};
+use serde::{Deserialize, Serialize};
 
 use crate::util::{get_matrix, gha_output, gha_print, CiTarget, ImageTarget};
 
 pub(crate) fn run(message: String, author: String) -> Result<(), color_eyre::Report> {
     let mut matrix: Vec<CiTarget> = get_matrix().clone();
-    if author == "bors[bot]" && message.starts_with("Try #") {
-        if let Some((_, args)) = message.split_once(": ") {
+    let pr = if author == "bors[bot]" && message.starts_with("Try #") {
+        let (label, args) = message
+            .split_once(':')
+            .ok_or_else(|| eyre::eyre!("bors message must start with \"Try #:\""))?;
+        let args = args.trim_start();
+        if !args.is_empty() {
             let app = TargetMatrixArgs::parse_from(args.split(' '));
             app.filter(&mut matrix);
         }
+        Some(
+            label
+                .strip_prefix("Try #")
+                .expect("commit message have \"Try #\" prefix")
+                .parse()?,
+        )
     } else {
```

Comment:
on 

> bors r+

this would be 

> Merge #PR

it can also be

> Merge #PR1 #PR2

https://github.com/cross-rs/cross/commit/5c96126a4625f95480ba3a1002c3fc4c2723f05c

Reply:
for multiple prs in one merge, checking all of them and then seeing if any pr do not have `no changelog` should work well